src/common/deleteButton/testUtils.ts

```diff
@@ -0,0 +1,10 @@
+import { ReactWrapper } from 'enzyme';
+
+import DeleteButton from './DeleteButton';
+import Button from '../button/Button';
+import ConfirmationModal from '../confirmationModal/ConfirmationModal';
+
+export const handleDelete = (wrapperContainingDeleteButton: ReactWrapper) => {
+  wrapperContainingDeleteButton.find(DeleteButton).find(Button).simulate('click');
+  wrapperContainingDeleteButton.find(ConfirmationModal).find(Button).last().simulate('click');
+};
```

Comment:
Should we add `"!**/testUtils.ts"` to `package.json jest.collectCoverageFrom` and `"src/**/testUtils.ts"` to `tsconfig.json exclude`?

Reply:
Decided to remove this as it's not really discoverable. Not worth the configuration changes for a one-off function IMO.